### PR TITLE
Cascading v8 to main

### PR DESCRIPTION
**Pull request by matthieu-crouzet**

### CODE REVIEW

**Review by mrednic-1A — DISMISSED** [review]

**Review by mrednic-1A — APPROVED** [review]

**Review by kpanot — APPROVED** [review]

File: packages/@ama-sdk/generator-sdk/src/generators/java-client-core/templates/swagger-codegen-java-client/src/main/resources/javaClient/settings.mustache [hunk 1]
```diff
@@ -3,17 +3,17 @@
...
+      <mirrorOf>des-sdks</mirrorOf>
     </mirror>
   </mirrors>
   <servers>
     <server>
-      <id>public-artifactory</id>
-      <username>${env.AR_USER}</username>
-      <password>${env.AR_PASSWORD}</password>
+      <id>des-sdks</id>
+      <username>AmadeusDigitalAirline</username>
```
Comment (kpanot) [inline packages/@ama-sdk/generator-sdk/src/generators/java-client-core/templates/swagger-codegen-java-client/src/main/resources/javaClient/settings.mustache:15]: Why using static string instead of variable?

File: packages/@ama-sdk/generator-sdk/project.json [hunk 2]
```diff
@@ -40,7 +40,9 @@
           "packages/@ama-sdk/generator-sdk/src/**/swagger-codegen-cli.jar",
           "packages/@ama-sdk/generator-sdk/src/**/config/*.json",
           "packages/@ama-sdk/generator-sdk/src/generators/**/target/*.jar",
-          "packages/@ama-sdk/generator-sdk/src/generators/*/templates/**/*",
+          "packages/@ama-sdk/generator-sdk/src/generators/shell/templates/**/*",
```
Comment (kpanot) [inline packages/@ama-sdk/generator-sdk/project.json:43]: Why this change, the `*` was not doing the job?

File: packages/@o3r/stylelint-plugin/src/public_api.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import rulesPlugins from './index';
+// Rules plugins have to be exported as default
+export default rulesPlugins;
```
Comment (kpanot) [inline packages/@o3r/stylelint-plugin/src/public_api.ts:3]: missing additional empty line

**Review by kpanot — APPROVED** [review]

File: packages/@ama-sdk/generator-sdk/project.json (hunk 2 above)
Comment (mrednic-1A) [reply packages/@ama-sdk/generator-sdk/project.json:43]: The `generators/*/templates` allowed in the packaging all the files inside `templates` folder for all the generators, not only the templates files for schematics generator. 
This was a problem for the packaging of java part of generators, because along with the `.jar` files in `target` folder we got also the `.class` files, which break the generators at runtime because it is trying to execute the them.
Ex: `generator-sdk/src/generators/java-client-core/templates/target` has matched with the `/generators/*/templates/**/*`, and that's why I specified exactly which are the `schematics` tempaltes to package.

File: packages/@ama-sdk/generator-sdk/src/generators/java-client-core/templates/swagger-codegen-java-client/src/main/resources/javaClient/settings.mustache (hunk 1 above)
Comment (matthieu-crouzet) [reply packages/@ama-sdk/generator-sdk/src/generators/java-client-core/templates/swagger-codegen-java-client/src/main/resources/javaClient/settings.mustache:15]: It's just the cascading 
maybe we need to fix it on v8 if you are not agree with what @mrednic-1A has done

File: packages/@o3r/stylelint-plugin/src/public_api.ts (hunk 3 above)
Comment (matthieu-crouzet) [reply packages/@o3r/stylelint-plugin/src/public_api.ts:3]: in fact there is one 
else we would see the red minus sign in a round

**Review by kpanot — APPROVED** [review]

File: packages/@ama-sdk/generator-sdk/src/generators/java-client-core/templates/swagger-codegen-java-client/src/main/resources/javaClient/settings.mustache (hunk 1 above)
Comment (mrednic-1A) [reply packages/@ama-sdk/generator-sdk/src/generators/java-client-core/templates/swagger-codegen-java-client/src/main/resources/javaClient/settings.mustache:15]: It's coming from a fix done when passing from jfrog to azure. we can redo it  8.0 if needed

**Review by mrednic-1A — APPROVED** [review]
